jupyter_server/services/kernels/kernelmanager.py

```diff
@@ -845,6 +848,30 @@ async def wrapped_method(self, *args, **kwargs):
 class ServerKernelManager(AsyncIOLoopKernelManager):
     """A server-specific kernel manager."""
 
+    # Override parent traits to make them configurable
+    client_class = DottedObjectName(
```

Comment:
shouldn't this be on the base KernelManager class in jupyter-client, so we inherit the configurability, rather than only here?